.github/workflows/init.yml

```diff
@@ -125,3 +131,40 @@ jobs:
               echo "support-packages-index-bucket=${SUPPORT_PACKAGES_BUCKET}"
             } >> "$GITHUB_OUTPUT"
           fi
+
+  downstream-check:
+    runs-on: ubuntu-latest
+    needs:
+    - common-variables
+    if: github.event_name == 'push' && (github.ref_type == 'tag' || startsWith(github.ref_name, 'release-'))
+    steps:
+      - uses: actions/checkout@v3
+      - name: Ensure downstream drivers exist for releases
+        run: |
+          DRIVER_VERSION="$(cat ${{ github.workspace }}/kernel-modules/MODULE_VERSION)"
+          DRIVER_COUNT="$(gsutil ls "gs://${{ needs.common-variables.outputs.cpaas-drivers-bucket }}/${DRIVER_VERSION}/" | wc -l)"
```

Comment:
There should probably be some error handling in case the bucket doesn't exist. Also I am not sure that this won't return an error if there are no files in the bucket. `grep asdf some_file.txt | wc -l` returns an error if asdf doesn't appear in the file. Might this have the same issue?

Reply:
I tested this, if the bucket doesn't exist an error message is printed to stderr and no lines are counted in stdout, so the check for 0 is still valid.